Makefile

```diff
@@ -36,7 +36,7 @@ build-all: proto-lint proto-gen update-swagger-docs build
 ### Analyzing
 
 lint:
-	GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
+	#GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
This makes lint ci fail?

Reply:
I will rollback!
Thank you :)

Reply:
My WSL was slow to respond so I didn't see the commented out. T_T

Reply:
So sad..